app/javascript/vue/label_template/insert_field_dropdown.vue

```diff
@@ -135,6 +146,14 @@
       });
     },
     methods: {
+      insertTag(field) {
```

Comment:
keep the naming consistent - if it emits an 'insertField' event, it should probably be called 'insertField'